StreamChat.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,61 +1,59 @@
 {
-  "object": {
-    "pins": [
-      {
-        "package": "Atlantis",
-        "repositoryURL": "https://github.com/ProxymanApp/atlantis",
-        "state": {
-          "branch": null,
-          "revision": "5e848072d45e3969bd1d69fab3b6c2151f826c41",
-          "version": "1.15.0"
-        }
-      },
-      {
-        "package": "Difference",
-        "repositoryURL": "https://github.com/krzysztofzablocki/Difference.git",
-        "state": {
-          "branch": null,
-          "revision": "02fe1111edc8318c4f8a0da96336fcbcc201f38b",
-          "version": "1.0.1"
-        }
-      },
-      {
-        "package": "Nuke",
-        "repositoryURL": "https://github.com/kean/Nuke",
-        "state": {
-          "branch": null,
-          "revision": "c0b32b0d7154afceb3fc08b199492f125ff4267e",
-          "version": "10.3.3"
-        }
-      },
-      {
-        "package": "StreamChatTestHelpers",
-        "repositoryURL": "https://github.com/GetStream/stream-chat-swift-test-helpers.git",
-        "state": {
-          "branch": null,
-          "revision": "577aedb1af8924efd2015d2da7afb7757fd27f78",
-          "version": "0.1.0"
-        }
-      },
-      {
-        "package": "SnapshotTesting",
-        "repositoryURL": "https://github.com/pointfreeco/swift-snapshot-testing.git",
-        "state": {
-          "branch": null,
-          "revision": "f8a9c997c3c1dab4e216a8ec9014e23144cbab37",
-          "version": "1.9.0"
-        }
-      },
-      {
-        "package": "Swifter",
-        "repositoryURL": "https://github.com/httpswift/swifter.git",
-        "state": {
-          "branch": null,
-          "revision": "9483a5d459b45c3ffd059f7b55f9638e268632fd",
-          "version": "1.5.0"
-        }
```

Comment:
Why are we updating the dependencies? 🤔

Reply:
json is unordered dictionary. I believe what happened here is that the `Package.resolved` got regenerated.

Reply:
The new JSON has Atlantis 1.16.0, so it looks like we updated Atlantis from 1.15.0 -> 1.16.0. But I think it is fine though 👍